lib/ttnt/test_selector.rb

```diff
@@ -50,6 +42,33 @@ def select_tests
 
     private
 
+    def test_to_code_mapping
```

Comment:
You are right, we don't have other "mapping"s around. Renamed to `#mapping`: https://github.com/Genki-S/ttnt/commit/3a400be86b9e9693accec7fb7a1435bb3fc1804d